### PR TITLE
Disabled the remove and add volunteer buttons after click

### PR DESCRIPTION
**Pull request by bledsoef**
Fixes issue #562 
Disabled the remove and add volunteer buttons after click.

### CODE REVIEW

File: app/static/js/trackVolunteers.js [hunk 1]
```diff
@@ -68,6 +69,7 @@ $(document).ready(function() {
   });
 
   $(".removeVolunteer").on("click", function() {
+    $(".removeVolunteer").prop("disabled", true)
```
Comment (BrianRamsay) [inline app/static/js/trackVolunteers.js]: Make sure if there is an error and the page doesn't reload that we can click the button again

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/static/js/trackVolunteers.js [hunk 2]
```diff
@@ -68,12 +69,14 @@ $(document).ready(function() {
...
     let username =  this.id;
     let eventId = $('#eventID').val()
     $.ajax({
       url: `/removeVolunteerFromEvent/${username}/${eventId}`,
       type: "POST",
       success: function(s) {
+         $("#selectVolunteerButton").prop("disabled", true)
```
Comment (BrianRamsay) [inline app/static/js/trackVolunteers.js]: This isn't the right button for here.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/static/js/trackVolunteers.js (hunk 1 above)
Comment (Boorclark) [reply app/static/js/trackVolunteers.js]: resolved

File: app/static/js/trackVolunteers.js (hunk 2 above)
Comment (Boorclark) [reply app/static/js/trackVolunteers.js]: resolved

File: app/static/js/trackVolunteers.js [hunk 3]
```diff
@@ -74,6 +75,7 @@ $(document).ready(function() {
       url: `/removeVolunteerFromEvent/${username}/${eventId}`,
       type: "POST",
       success: function(s) {
+         $(".removeVolunteer").prop("disabled", true)
```
Comment (BrianRamsay) [inline app/static/js/trackVolunteers.js]: This disable is unnecessary, as the page is about to reload (and you already disabled it on click anyway). What you need is to re-enable the button if you don't reload the page

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/static/js/trackVolunteers.js (hunk 3 above)
Comment (bledsoef) [reply app/static/js/trackVolunteers.js]: resolved

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**